messages/definitions.json

```diff
@@ -30,9 +30,21 @@
       ],
       "properties": {
         "@context": {
-          "type": "string",
-          "title": "Device context",
-          "description": "Context of the device"
+          "oneOf": [
+            {
+              "type": "string",
+              "title": "Device context",
+              "description": "JSON-LD context used by the device"
+            },
+            {
+              "type": "array",
+              "items": {
+                "type": "string",
+                "title": "Device contexts",
+                "description": "An array of JSON-LD contexts used by the device"
+              }
+            }
+          ]
```

Comment:
With this change we can support these two context types: `@context : "URL"` and `@context : ["url", "url"]`

What about: 
```
@context : [ "url", { myprefix : "url"}] 
```

Reply:
Ah OK, so we need something more like this? https://github.com/w3c/wot-scripting-api/blob/main/typescript/thing-description/schema/td-json-schema-validation.json#L70

Reply:
yes, I think so.

Reply:
How's this?